checksum/src/main/resources/application.conf

```diff
@@ -0,0 +1,16 @@
+sqs {
```

Comment:
Should there be conf for the environments, intg, staging, prod, or is it too early for this?

Reply:
The actual values in the lambda are set using the environment variables in the lambda which are set by terraform, so we always set them to the environment specific ones for the lambda in each environment.